spec/services/algolia/vacancy_alert_builder_spec.rb

```diff
@@ -10,7 +10,7 @@
   let(:default_radius) { 10 }
   let(:date_today) { Time.zone.today.to_datetime }
   let(:location_point_coordinates) { Geocoder::DEFAULT_STUB_COORDINATES }
-  let(:location_radius) { subject.convert_radius_in_miles_to_metres(default_radius) }
+  let(:location_radius) { (default_radius * 1.60934 * 1000).to_i }
```

Comment:
I have read `* 1.60934 * 1000` many times, can we define a costant for that?

Reply:
Haha yes of course